### PR TITLE
Remove jQuery from the citation modal (except fomantic)

### PR DESCRIPTION
**Pull request by yardenshoham**
- Switched to plain JavaScript
- Tested the citation modal functionality and it works as before

# Demo using JavaScript without jQuery
demo

### CODE REVIEW

File: web_src/js/features/citation.js [hunk 1]
```diff
@@ -14,56 +15,57 @@ async function initInputCitationValue($citationCopyApa, $citationCopyBibtex) {
...
-  if ((!$citationCopyApa.length && !$citationCopyBibtex.length) || !$inputContent.length) return;
   const updateUi = () => {
     const isBibtex = (localStorage.getItem('citation-copy-format') || defaultCitationFormat) === 'bibtex';
-    const copyContent = (isBibtex ? $citationCopyBibtex : $citationCopyApa).attr('data-text');
-
-    $inputContent.val(copyContent);
-    $citationCopyBibtex.toggleClass('primary', isBibtex);
-    $citationCopyApa.toggleClass('primary', !isBibtex);
+    const copyContent = (isBibtex ? citationCopyBibtex : citationCopyApa).getAttribute('data-text');
+    inputContent.setAttribute('value', copyContent);
```
Comment (silverwind) [inline web_src/js/features/citation.js]:
```suggestion
    inputContent.value = copyContent;
```
I think this is more common throughout the codebase to set value.

File: web_src/js/features/citation.js [hunk 2]
```diff
@@ -14,56 +15,57 @@ async function initInputCitationValue($citationCopyApa, $citationCopyBibtex) {
...
+
+      citationCopyBibtex.addEventListener('click', () => {
         localStorage.setItem('citation-copy-format', 'bibtex');
         updateUi();
       });
 
-      $inputContent.on('click', () => {
-        $inputContent.trigger('select');
+      inputContent.addEventListener('click', () => {
+        inputContent.select();
```
Comment (silverwind) [inline web_src/js/features/citation.js:68]: Is it the same outcome? If not do `inputContent.dispatchEvent(new Event('select'));`

Comment (yardenshoham) [reply web_src/js/features/citation.js:68]: I tested. It's the same.

Comment (silverwind) [reply web_src/js/features/citation.js:68]: Ok keep it.

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Pull request merged**